tests/lib/rules/no-unused-prop-types.js

```diff
@@ -4376,7 +4376,7 @@ ruleTester.run('no-unused-prop-types', rule, {
         line: 11,
         column: 8
       }]
-    }, { // None of the props are used issue #1162
+    }, {// None of the props are used issue #1162
```

Comment:
```suggestion
    }, { // None of the props are used issue #1162
```

Reply:
This is changed by eslint@6. Without this change the pretest will fail.

Reply:
what lint rule complains about this?

Reply:
object-curly-spacing

Reply:
```suggestion
    }, {
      // None of the props are used issue #1162
```
this, then ^